src/many-ledger/src/storage.rs

```diff
@@ -171,6 +181,45 @@ impl LedgerStorage {
         self.latest_tid.clone()
     }
 
+    pub fn create_snapshot(&self, height: u64) -> Result<(), ManyError> {
```

Comment:
Is there a way to reload a snapshot in the system?

If so, can you document it?

If not, we would probably need that, WDYT?

Reply:
Merk does not support this, so the way we'll load snapshots is to close merk, delete the files, untargz the snapshot and then reopen Merk.